compel/plugins/Makefile

```diff
@@ -16,7 +16,12 @@ asflags-y		+= -I compel/include/uapi
 
 # General compel includes
 ccflags-y		+= -iquote compel/include
+
+ifeq ($(ARCH),mips)
+ccflags-y		+= -mno-abicalls -fno-stack-protector
+else
 ccflags-y		+= -fpie -fno-stack-protector
+endif
```

Comment:
If it is possible to build MIPS variant with `-fpie`, please only add `-mno-abicalls` for MIPS, e.g. something like:
```
ccflags-y		+= -fpie -fno-stack-protector
ifeq ($(ARCH),mips)
ccflags-y		+= -mno-abicalls
endif
```

Reply:
MIPS can not use -fpie or -fpic when used -mno-abicalls